src/QueryOperators.jl

```diff
@@ -5,8 +5,10 @@ using IteratorInterfaceExtensions
 using TableShowUtils
 
 export Grouping, key
+export select, remove, rename
```

Comment:
And lets not export these. We might later, but for now lets keep this API more internal.